app/test_main.py

```
@@ -1,3 +1,19 @@
 from app.main import get_human_age
+import pytest
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected",
+    [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 27, [2, 2]),
+        (28, 28, [3, 2]),
+        (100, 100, [21, 17]),
+    ]
+)
```

Great job using `@pytest.mark.parametrize` to create a concise set of tests. The current test cases correctly verify the examples provided in the task description.

However, the tests are missing some edge cases required by the checklist. This violates checklist item #2: "Make sure you've tests checks all edged situations such as: ... if function resive data out of normal range, such as negative numbers... if the function receives an incorrect type of data it raises the correct exception."

Specifically, you should consider adding tests for:
- **Negative numbers**: What should be the result for `get_human_age(-5, -10)`?
- **Incorrect data types**: The function should raise an exception for non-integer inputs like `get_human_age('a', 'b')`. You can use `pytest.raises` to test this.